pokemongo_bot/cell_workers/evolve_all_worker.py

```diff
@@ -109,7 +109,7 @@ def _release_evolved(self, release_cand_list_ids):
                     # Transfering Pokemon
                     self.transfer_pokemon(pokemon_id)
                     logger.log(
-                        '[#] {} has been exchanged for candy!'.format(pokemon_name), 'green')
+                        '[#] {} has been exchanged for candy!'.format(pokemon_name), 'red')
```

Comment:
Why this change? Red means bad, green means positive, this shouldn't be changed.